src/app/App.js

```diff
@@ -47,4 +48,17 @@ export const App = ({ options }) => {
     modeHall.hidden = !modeHall.hidden;
     modeRules.hidden = !modeRules.hidden;
   }
+  
+  // dynamicaly set active game mode
+  gameModes.forEach(mode => {
+    mode.addEventListener('click', (e) => {
+      gameModes.forEach(mode => {
```

Comment:
zmieniłbym gdzieś nazwę bo teraz parametr z 53 linii nazywa się tak samo jak ten z 55 co może być mylące

Reply:
hm, czy tu czy tu to dalej mode ale rozumiem problem. Jakas sugestia?

Reply:
Ja osobiście nie widzę problemu, choć wiem że niektóre narzędzia do analizy kodu potrafią się o to doczepić. Jeśli bardzo to wam przeszkadza to można sobie zamknąć `     gameModes.forEach((mode) => {
        mode.classList.remove('menu__item--selected');
      });` w ramach jakieś np.: deselectAllGameModes i tu ją tylko wywołać, tak cały handler będzie bardziej czytelny